src/avatar/stories/Avatar.stories.tsx

```diff
@@ -1,154 +1,130 @@
-export const NameAndFallback = base({
-  name: "Anurag Hazra",
-  fallback: <ClockIcon />,
+export const OnlineBadge = base({
+  src: "https://bit.ly/ryan-florence",
+  name: "Ryan Florence",
+  status: "online",
+});
+export const SleepBadge = base({
+  src: "https://bit.ly/ryan-florence",
+  name: "Ryan Florence",
+  status: "sleep",
+});
+export const TypingBadge = base({
+  src: "https://bit.ly/ryan-florence",
+  name: "Ryan Florence",
+  status: "typing",
 });
 
-export const WithBadge: Story<AvatarProps> = () => {
-  const [isTyping, setTyping] = React.useState(false);
-
-  return (
-    <>
-      <Avatar size="xl" src="https://bit.ly/dan-abramov">
-        <AvatarBadge position="bottom-right">
-          {isTyping ? <TypingAnimation /> : <OnlineDot />}
-        </AvatarBadge>
-      </Avatar>
-
-      <br />
+export const BadgeExample = storyTemplate<AvatarProps>(
+  args => {
+    const [isTyping, setTyping] = React.useState(false);
 
-      <label>
-        isTyping?
-        <input
-          type="checkbox"
-          name="typing"
-          onChange={() => setTyping(!isTyping)}
+    return (
+      <>
+        <Avatar
+          status={isTyping ? "typing" : "online"}
+          className="flex"
+          {...args}
         />
-      </label>
-    </>
-  );
-};
-
-export const Statuses: Story<AvatarProps> = () => (
-  <AvatarGroup size="xl">
-    <Avatar size="xl" src="https://bit.ly/dan-abramov" name="Anurag Hazra">
-      <AvatarBadge>
-        <Status status="online" />
-      </AvatarBadge>
-    </Avatar>
-    <Avatar size="xl" src="https://bit.ly/dan-abramov" name="Anurag Hazra">
-      <AvatarBadge>
-        <Status status="offline" />
-      </AvatarBadge>
-    </Avatar>
-    <Avatar size="xl" src="https://bit.ly/dan-abramov" name="Anurag Hazra">
-      <AvatarBadge>
-        <Status status="sleep" />
-      </AvatarBadge>
-    </Avatar>
-  </AvatarGroup>
-);
-
-export const OnlineTooltip: Story<AvatarProps> = () => {
-  const tooltip = useTooltipState({ placement: "right-start", gutter: 5 });
+        <label className="block mt-4">
+          isTyping?
+          <input
+            type="checkbox"
+            name="typing"
+            className="ml-2"
+            onChange={() => setTyping(!isTyping)}
+          />
+        </label>
+      </>
+    );
+  },
+  { size: "xl", src: "https://bit.ly/dan-abramov" },
+)({});
+
+export const Group = storyTemplate<AvatarProps>(
+  args => (
+    <AvatarGroup {...args}>
+      <Avatar
+        src="https://bit.ly/dan-abramov"
+        name="Dan Abramov"
+        status="online"
+      />
+      <Avatar
+        src="https://bit.ly/kent-c-dodds"
+        name="Kent Dodds"
+        status="sleep"
+      />
+      <Avatar
+        src="https://bit.ly/code-beast"
+        name="Code Beast"
+        status="typing"
+      />
+    </AvatarGroup>
+  ),
+  { size: "xl" },
+)({});
 
+export const GroupWithLimit = () => {
```

Comment:
would be nice to use `storyTemplate<AvatarGroupProps>` here and pass the params from the base() so we can change them dynamically

Reply:
I thought but left so that we can see all sizes without changing in controls

Reply:
Yeah don't need the size, just for the other props, border, borderColor, showBorder